openedx/core/djangoapps/programs/tasks.py

```diff
@@ -77,22 +78,28 @@ def get_inverted_programs(student):
     return inverted_programs
 
 
-def get_certified_programs(student):
+def get_certified_programs(student: User, raise_on_error: bool = False) -> List[str]:
     """
     Find the UUIDs of all the programs for which the student has already been awarded
     a certificate.
 
     Args:
-        student:
-            User object representing the student
+        student: User object representing the student
+
+    Keyword Arguments:
+        raise_on_error (bool): Reraise errors back to the caller, instead if returning empty results.
 
     Returns:
         str[]: UUIDs of the programs for which the student has been awarded a certificate
 
     """
     certified_programs = []
-    for credential in get_credentials(student, credential_type='program'):
-        certified_programs.append(credential['credential']['program_uuid'])
+    for credential in get_credentials(
+        student,
+        credential_type="program",
```

Comment:
Should this be using that `PROGRAM_CERTIFICATE` constant at the top of the file?